generating_an_expression_matrix/README.md

```diff
@@ -61,98 +61,120 @@ This is what the your history will look like:
 
 ![concatenating annotation files](concatenate.png)
 
-Select biological and technical sequences from your history, and use them as inputs. Run the tool and generate your combined annotations.
- 
-> EXERCISE: Which of the 'attributes' in the last column of the GTF files contains the transcript and gene identifiers?
+Select biological and spike-in sequences from your history, and use them as inputs. Run the tool and generate your combined annotations.  
+> EXERCISE: Which of the 'attributes' in the last column of the GTF files contains the transcript and gene identifiers? Hint: The file is organised such that the last column (headed 'Group') contains a wealth of information in the format: attribute1 "information associated with attribute";attribute2 "information associated with attribute 2" etc.
 
-Now we have combined annotations, we can parse the GTF file using the [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html) package in R. We've made a convenience tool in Galaxy to wrap that functionality for you- search for 'GTF2GeneList':
+Now we have combined annotations, we can parse the GTF file using the [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html) package in R. This parsing will give us an index with a list of transcript names and their corresponding gene names for counting. We've made a convenience tool in Galaxy to wrap that functionality for you- search for 'GTF2GeneList':
 
-![GTF2GeneList](gtf2genelist.png)
+![GTF2GeneList](gtf2genelist2.png)
 
 This tool is designed to do a variety of things. For our purposes parameterise like:
 
- * "Feature type..." : transcript
- * "Field to place first..." : (use the transcript ID field you identified above)
+ * "Feature type..." : transcript        
+ * "Field to place first..." : (use the attribute you identified above for the transcript)
  * "Suppress header" : yes
- * "Comma separated list of field names" : (transcript ID field),(gene ID field)
- * "Append version to transcript identifiers" : yes 
+ * "Comma separated list of field names" : (transcript attribute),(gene attribute)
+ * "Append version to transcript identifiers" : yes
  * "Flag mitochondrial features" : no
- * "Filter a FASTA": no
 
-> EXERCISE: compare the transcript identifiers you'll find in the FASTA file you were given at the start with the transcript ID values in the GTF, to see why we say 'yes' to the 'transcript identifiers' question
+> EXERCISE: compare the transcript identifiers you'll find in the FASTA file you were given at the start with the transcript ID values in the GTF, to see why we say 'yes' to the 'transcript identifiers' question. Note that 'version' here refers to the number of times the reference has been edited.
 
 ### 3. Running quantification
 
 Now we have everything we need to run Alevin:
 
- * A Salmon index for our transcriptome
+ * A Salmon index for our transcriptome (transcript name or identifier & sequence)
```

Comment:
No, index here is the actual binary index structure prepared by Salmon - your adendum is a little confusing I'm afraid.

Reply:
OK, so we actually don't have that yet. What we have is the sequence list, which will then be turned into an index by Salmon?

Reply:
Well, this version assumes that a pre-built index is present. If not, we'll have to make a few edits throughout. 

Otherwise the input is the transcriptome sequence itself, not an index.